src/seedu/addressbook/data/person/Address.java

```diff
@@ -52,6 +52,11 @@ public boolean equals(Object other) {
     public int hashCode() {
         return value.hashCode();
     }
+    
+    @Override
+    public String getPrintableString(){
```

Comment:
Correct implementation of the interface.
As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.